### ISSUE TITLE
Abnormal grip

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#5908]
**Preferred term label:** Abnormal grip

**Synonyms**

**Definition (free text, please give PubMed ID)** An abnormality handgrip including weak grip and/or handgrip myotonia., PMID:26055647

**Parent term (use hpo.jax.org/app)** Abnormality of the hand HP:0001155

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
Adding as Weak grip
